packages/tree-widget/src/components/trees/CategoriesTree.tsx

```diff
@@ -1,19 +1,24 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
 import {
   IModelApp,
   IModelConnection,
   Viewport,
 } from "@bentley/imodeljs-frontend";
-import { CategoryTree, toggleAllCategories } from "@bentley/ui-framework";
+import {
+  CategoryTree,
+  toggleAllCategories,
+  getCategories,
+} from "@bentley/ui-framework";
 import { IconButton } from "../IconButton";
 import { SearchBar } from "../search-bar/SearchBar";
 import { useTreeFilteringState } from "../TreeFilteringState";
 import "./CategoriesTree.scss";
 import { TreeWidget } from "../../TreeWidget";
+import { CategoryVisibilityHandler } from "@bentley/ui-framework/lib/ui-framework/imodel-components/category-tree/CategoryVisibilityHandler";
```

Comment:
Could you import `CategoryVisibilityHandler` from the root level barrel file in ui-framework, so just add it to the existing imports from ui-framework

Reply:
Nope, is internal atm, would need to update core first.

Reply:
damn, thats unfortunate.